golem-common/src/golem_version.rs

```diff
@@ -0,0 +1,12 @@
+pub use git_version::git_version;
+
+#[macro_export]
+macro_rules! golem_version {
+    () => {
+        golem_common::golem_version::git_version!(
+            args = ["--tags", "--abbrev=0"],
+            cargo_prefix = "",
+            fallback = "0.0.0"
```

Comment:
should not be fallback 1.0.0 (as we have 1.0.+ released)

Reply:
i would keep it as "not known", given:
 - when git cloned repo is used, we can assume git is installed and git describe will work
 - when using cargo install it should use the CARGO_PKG_VERSION fallback (the cargo_prefix arg triggers this, but we should test this)
 
so in general the fallback is a "should not happen" value, but if it happens i would rather be able to see a "zero" value for debug, then assuming any version